db/migrate/20151231161821_create_divisions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDivisions < ActiveRecord::Migration
+  def change
+    create_table :divisions do |t|
+      t.references :organization, index: true, foreign_key: true
```

Comment:
<old man voice> Back in my day, we had to use foreigner for that </old man voice>